sharktank/tests/evaluate/perplexity_iree_test.py

```diff
@@ -34,6 +34,7 @@ def setUp(self):
         with open(self.baseline_perplexity_scores, "r") as f:
             self.baseline_perplexity = json.load(f)
 
+    @pytest.mark.xfail(reason="Compile Error")
```

Comment:
Compilation succeeds I think?

```suggestion
    @pytest.mark.xfail(reason="Runtime segfault")
```